ush/build.comgsi

```diff
@@ -5,45 +5,98 @@
+    GSILIBS="/scratch1/BMC/comgsi/precompiled/GSILIBS/b_intel18.0.5.274_impi2018.0.4"
+
+################# Jet ####################
+elif [[ -d /jetmon ]] ; then
+    source /etc/profile.d/modules.sh
     modulefile="/home/rtrr/PARM_EXEC/modulefiles/modulefile.jet.GSI_UPP_WRF"
-    export G2_LIB4="/lfs4/BMC/wrfruc/gge/precompiled/WPSV3.9/ungrib/src/ngl/g2/libg2_4.a"
-    export G2TMPL_LIB="/lfs4/BMC/wrfruc/gge/precompiled/comupp/lib/libg2tmpl.a"
-    export GRIB2INC="/lfs4/BMC/wrfruc/gge/precompiled/comupp/include"
-elif [[ -d /glade ]] ; then  ### cheyenne
+    NCEPLIBS="/lfs4/BMC/wrfruc/gge/precompiled/NCEPLIBS/b_intel18.0.5.274_impi2018.4.274/install"
+    GSILIBS="/lfs4/BMC/wrfruc/gge/precompiled/GSILIBS/b_intel18.0.5.274_impi2018.4.274"
+
+################# Cheyenne ####################
+elif [[ -d /glade ]] ; then
+    source /etc/profile.d/modules.sh
     modulefile="/glade/p/ral/jntp/gge/modulefiles/modulefile.cheyenne.GSI_UPP_WRF"
+    NCEPLIBS="/glade/p/ral/jntp/gge/precompiled/NCEPLIBS/b_intel18.0.5_impi2018.4.274/install"
+    GSILIBS="/glade/p/ral/jntp/gge/precompiled/GSILIBS/b_intel18.0.5_impi2018.4.274"
+    #modulefile="/glade/p/ral/jntp/gge/precompiled/NCEPLIBS/modulefile.cheyenne.GSI_UPP_WRF.gnu"
+    #NCEPLIBS="/glade/p/ral/jntp/gge/precompiled/NCEPLIBS/b_gnu8.3.0_openmpi3.1.4/install"
+    #GSILIBS="/glade/p/ral/jntp/gge/precompiled/GSILIBS/b_gnu8.3.0_openmpi3.1.4/"
+
+################# Orion ####################
+elif [[ -d /work/noaa ]] ; then  ### orion
+    modulefile="/home/gge/modulefiles/modulefile.orion.GSI_UPP_WRF"
+    NCEPLIBS="/work/noaa/wrfruc/gge/precompiled/NCEPLIBS/b_intel2018.4_impi2018.4/install"
+    GSILIBS="/work/noaa/wrfruc/gge/precompiled/GSILIBS/b_intel2018.4_impi2018.4"
+
+################# STAMPEDE2 for GSI-CAPS branch ####################
+elif [[ "`grep -i "stampede2" /etc/hosts | head -n1`" != "" ]] ; then
+   # source /etc/profile.d/modules.sh
+   # modulefile="/my/modulefile.me.GSI_UPP_WRF"
+    NCEPLIBS="/work/06809/jpark217/stampede2/20200619/NCEPLIBS/build/install"
+    GSILIBS="/work/06809/jpark217/stampede2/GSILIBS/build"
+
+################# Generic ####################
 else
-    echo "unknown machine"
+    echo -e "\nunknown machine"
+    echo "Please modify build.comgsi at this location"
+    echo "to load required modules and setup NCEPLIBS and GSILIBS"
+    ##follow the above examples and delete the following "exit 9" to go forward
     exit 9
+    source /etc/profile.d/modules.sh
+    modulefile="/my/modulefile.me.GSI_UPP_WRF"
+    NCEPLIBS="/my/NCEPLIBS/b_intel18.0.5_impi2018.4.274/install"
+    GSILIBS="/my/GSILIBS/b_intel18.0.5_impi2018.4.274/"
 fi
 
-if [ ! -f $modulefile ]; then
-    echo "modulefiles $modulefile does not exist"
-    exit 10
-fi
-source $modulefile
+#if [ ! -f $modulefile ]; then
+#    echo "modulefiles $modulefile does not exist"
+#    exit 10
+#fi
+#source $modulefile
```

Comment:
why comment out these lines?  If Stampede does not need to source a module file, you can make a fake and simple one to make this script still there so that other platforms can use it without a problem.

Reply:
This was already modified at my second commit ("added module files for stampede2 (GSI-CAPS branch)"
92b66ab )